finetuner-workflow/finetune-workflow.yaml

```diff
@@ -251,6 +263,22 @@ spec:
             - name: model_path
               value: "finetunes/results-{{workflow.parameters.run_name}}"
         when: "{{workflow.parameters.run_inference}} == true"
+  - name: check-model
+    inputs:
+      parameters:
+        - name: model
+    retryStrategy:
+      limit: 1
+    container:
+      image: "alpine:3.12"
+      command: [ "/bin/sh", "-c" ]
+      args: [ "curl -I -XGET -s -o /dev/null -w \"%{http_code}\" https://accel-object.ord1.coreweave.com/tensorized/{{inputs.parameters.model}}/model.tensors | grep -q 200 && echo -n 'true' > /tmp/output.txt || echo -n 'false' > /tmp/output.txt" ]
```

Comment:
Does a `HEAD` not work instead of a `GET`? You're fetching the entire file here.

Reply:
The `-I` argument should be getting just fetches the header information. Running this takes a second to execute since it only fetches the header:
`curl -I -XGET -s -o /dev/null -w \"%{http_code}\" https://accel-object.ord1.coreweave.com/tensorized/EleutherAI/pythia-2.8b/model.tensors`
Whereas running the same command without the `-I` argument takes quite a lot longer, since it is actually fetching the whole file:
`curl -XGET -s -o /dev/null -w \"%{http_code}\" https://accel-object.ord1.coreweave.com/tensorized/EleutherAI/pythia-2.8b/model.tensors`
A more readable flag would be `--head` though lol.